src/tsn_adapters/flows/eps/real_time_flow.py

```
@@ -0,0 +1,234 @@
+
+
+def _is_published(tn_block: TNAccessBlock, stream_id: str, date_unix: int) -> bool:
+    """Return True if a record for this exact date already exists in TN."""
+    try:
+        df = tn_block.read_records(
+            stream_id=stream_id,
+            date_from=date_unix,
+            date_to=date_unix,
+        )
+        return not df.empty
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)